ftag/wps/working_points.py

```diff
@@ -218,7 +219,8 @@ def setup_common_parts(args):
     zprime_cuts = Cuts.from_list(args.zprime_cuts) + default_cuts
 
     # prepare to load jets
-    all_vars = next(iter(flavs)).cuts.variables
+    var_ls = [flav.cuts.variables for flav in flavs]
+    all_vars = list(set(chain(*var_ls)))
```

Comment:
I think something like 

```
all_vars = list(set(sum([flav.cuts.variables for flav in flavs], [])))
```

should work here, saving the import

Reply:
funnily this fails the linting check with "C419 - Unnecessary list comprehension", I think a bug since it does not realize sum here is taking two arguments, so the list comp can't actually be removed ?

Reply:
sorted using separate variables

Reply:
Ah that just means you should do 

```
all_vars = list(set(sum((flav.cuts.variables for flav in flavs), [])))
```

so using a generator expression rather than a list comprehension. Was there no autofix for that?